### PR TITLE
Spelling errors

### PR DESCRIPTION
**Pull request by hamishwillee**
This fixes many spelling errors in the codebase. These were primarily found by using codespell on the documentation and then doing a careful replacement on the source.

Note that there are many other typos in the source - I haven't fixed those that touch actual code.

For example, the correct SI unit is "degree Celsius" (yes, including the capital letter). However the `@unit` is celcius so I didn't change that.

### CODE REVIEW

File: src/drivers/uwb/uwb_sr150/module.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ parameters:
         UWB_INIT_OFF_X:
           description:
             short: UWB sensor X offset in body frame
-            long: UWB sensor positioning in relation to Drone in NED. X offset. A Positiv offset results in a Position o
+            long: UWB sensor positioning in relation to Drone in NED. X offset. A Positive offset results in a Position o
```
Comment (hamishwillee) [inline src/drivers/uwb/uwb_sr150/module.yaml:16]: Note, don't know how this sentence was supposed to end.

Comment (julianoes) [reply src/drivers/uwb/uwb_sr150/module.yaml:16]: Hmmm :thinking:

**Review by julianoes — APPROVED** [review]
What is it with ouput!? :joy: 

This was one of the most entertaining PR review!

P.S. Can you do one for MAVSDK too? But make sure to fix comments in proto, not in the auto-generated source.

File: src/drivers/uwb/uwb_sr150/module.yaml (hunk 1 above)
Comment (hamishwillee) [reply src/drivers/uwb/uwb_sr150/module.yaml:16]: Well, we're no worse off for this going in as is. Who might know the intent?

**Comment by hamishwillee** [comment]
I don't know what's up with output :-). There are a lot of incorrectly named variables, but that's not such a big deal as they aren't exposed.

Anyway, thanks for the review. Yes, I will look at MAVSDK now.

File: src/drivers/uwb/uwb_sr150/module.yaml (hunk 1 above)
Comment (hamishwillee) [reply src/drivers/uwb/uwb_sr150/module.yaml:16]: Closing. No worse than before.

**Pull request merged**